google/cloud/pubsub/internal/tracing_message_callback.cc

```diff
@@ -50,13 +53,17 @@ class TracingMessageCallback : public MessageCallback {
     auto span =
```

Comment:
So we would make a span when `enable_otel_ == false`? Odd.

Reply:
Resolved above.